cmd/wallet-js-sdk/src/user/wallet-user.js

```diff
@@ -189,6 +190,7 @@ export class WalletUser {
       controller = "",
       verificationMethod = "",
       proofType = "",
+      welcomeMessageDismissed = false,
```

Comment:
imo `skipWelcomeMsg` would be more concise.

Reply:
yes, some generic name, so that it can be used community for other purposes too.

Reply:
Should I make it optional as well @rolsonquadras @sudeshrshetty?

Reply:
it is optional already since u r setting default value, all fields I see here are optional.

Reply:
Yes, sure. I mean in the parameters description I'd add `(optional)` as well

Reply:
yes

Reply:
Updated